### PR TITLE
Move MCT component to TFX Addons

### PR DESCRIPTION
**Pull request by hanneshapke**
Addresses #[82](https://github.com/tensorflow/tfx-addons/issues/82)

- [x] Tests pass
- [x] Test example notebook
- [x] Update `CODEOWNER`

### CODE REVIEW

**Comment by hanneshapke** [comment]
@codesue Let's chat async what to do with https://github.com/tensorflow/model-card-toolkit/blob/master/model_card_toolkit/utils/tfx_util.py and the related tests. Are there reasons for it to stay in the MCT repo?

**Comment by hanneshapke** [comment]
I think it should probably stay. It is used in core and it seems to contain the connector pieces to the TFX libraries, not the component par se.

File: examples/model_card_generator/MLMD_Model_Card_Toolkit_Demo.ipynb
Comment (codesue) [inline examples/model_card_generator/MLMD_Model_Card_Toolkit_Demo.ipynb:40]: I believe MLMD is only used indirectly through TFX in this example. Consider removing the MLMD import and version check and removing mentions of MLMD wherever it makes sense to do so.

File: examples/model_card_generator/MLMD_Model_Card_Toolkit_Demo.ipynb
Comment (codesue) [inline examples/model_card_generator/MLMD_Model_Card_Toolkit_Demo.ipynb:1104]: Let's remove this unused import.

File: examples/model_card_generator/census_income_constants.py
Comment (codesue) [inline examples/model_card_generator/census_income_constants.py:1]: You were right about these files not being needed to run the example. I'm starting to think they might have been commited by mistake. We may want to consider adding them to the gitignore.

File: tfx_addons/model_card_generator/executor_test.py
Comment (codesue) [inline tfx_addons/model_card_generator/executor_test.py:149]: With https://github.com/tensorflow/model-card-toolkit/pull/251 merged, we can remove  `confidence_interval=model_card_pb2.ConfidenceInterval()` and `sensitive=model_card_pb2.SensitiveData()` below. See [example diff](https://github.com/tensorflow/model-card-toolkit/pull/251/commits/a515b86dca5817e6172fa9994614410448c368d6#diff-f7f73c3eb6f562cbe2ddb2770da515990e31a0a145e1c8c377fea59e5955566c).

File: examples/model_card_generator/census_income_constants.py
Comment (hanneshapke) [reply examples/model_card_generator/census_income_constants.py]: I removed those notebook generated files

File: tfx_addons/model_card_generator/executor_test.py
Comment (hanneshapke) [reply tfx_addons/model_card_generator/executor_test.py:149]: Thank you for the hint! I removed the references.

**Review by rcrowe-google — APPROVED** [review]
Thanks!
/lgtm

**Review by codesue — COMMENTED** [review]
lgtm after tests pass

File: .gitignore
Comment (hanneshapke) [reply .gitignore:44]: Good idea! Thank you!

File: CODEOWNERS
Comment (hanneshapke) [reply CODEOWNERS:66]: Arg, good catch.

File: .gitignore
Comment (codesue) [reply .gitignore:44]: Cool! Let's revert the changes to this file before merging.

**Review by codesue — APPROVED** [review]

**Comment by hanneshapke** [comment]
Thank you @codesue

**Pull request merged**
